src/main/java/discord4j/discordjson/json/ForumThreadMessageParamsData.java

```diff
@@ -0,0 +1,37 @@
+package discord4j.discordjson.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import discord4j.discordjson.Id;
+import discord4j.discordjson.possible.Possible;
+import org.immutables.value.Value;
+
+import java.util.List;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableForumThreadMessageParamsData.class)
+@JsonDeserialize(as = ImmutableForumThreadMessageParamsData.class)
+public interface ForumThreadMessageParamsData {
+
+    static ImmutableForumThreadMessageParamsData.Builder builder() {
+        return ImmutableForumThreadMessageParamsData.builder();
+    }
+
+    Possible<String> content();
+
+    Possible<List<EmbedData>> embeds();
+
+    @JsonProperty("allowed_mentions")
+    Possible<List<AllowedMentionsData>> allowedMentions();
+
+    Possible<List<ComponentData>> components();
+
+    @JsonProperty("sticker_ids")
+    Possible<List<Id>> stickerIds();
+
+    Possible<List<PartialAttachmentData>> attachments();
```

Comment:
Based in the class about messages i think the attachments things is handled by the Discord4J project and not here.
unless this class is for the get things and not the request.
not really sure but is what i found here https://github.com/Discord4J/Discord4J/blob/5d71d92fcb04b7e526d90f2e1f143ee08fb6ff19/rest/src/main/java/discord4j/rest/entity/RestChannel.java

Reply:
Resolved in 4e2c9e9b767324d7bbd6b51cf60d3611d0e57f61